static/app/components/tooltip.tsx

```diff
@@ -234,13 +242,31 @@ export function DO_NOT_USE_TOOLTIP({
       (skipWrapper || typeof triggerChildren.type === 'string')
     ) {
       // Basic DOM nodes can be cloned and have more props applied.
-      return cloneElement(triggerChildren, {...containerProps, ref: setRef});
+      return (
+        <ClassNames>
+          {({cx, css}) =>
+            cloneElement(triggerChildren, {
+              ...containerProps,
+              className: cx(
+                showUnderline && css(theme.tooltipUnderline),
+                triggerChildren.props.className
+              ),
+              ref: setRef,
+            })
+          }
+        </ClassNames>
+      );
```

Comment:
ooof, this is kinda gross, we're wrapping _everything_ with a `ClassNames` that uses tooltip now, even when we're _not_ setting `showUnderline`

Reply:
Instead of using the ClassNames utility, would you be ok just setting the `style` property?

Reply:
hmm because `theme.tooltipUnderline` is a return value of emotion's `css` function, it's not directly passable to the style prop. Lemme see if there's a cleaner way to do this.

Reply:
Does it need to be a `css` function?

Reply:
okay I deleted the `css` function call in theme, so `tooltipIndicator` will now just return a style object that can be passed into the `style` prop - https://github.com/getsentry/sentry/pull/33746/commits/184d9c47b8ae1993b14876a07e2fa93a033f777c. Originally decided to use the `css` function cause it provides better type checking, but it's not really necessary.